marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/InitializedDataHubProject.java

```diff
@@ -0,0 +1,96 @@
+package com.marklogic.hub.hubcentral;
+
+import com.marklogic.hub.HubClient;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class InitializedDataHubProject extends AllArtifactsProject{
```

Comment:
I don't think this class is needed - the methods in here can just be private to the one test class that uses it. The name is a little confusing because it implies that it's a newly initialized project, but it's more than that - it's all the stuff from the "all-artifacts" project too.